src/test/java/com/exasol/testutil/ExceptionAssertions.java

```diff
@@ -0,0 +1,42 @@
+package com.exasol.testutil;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.hamcrest.Matcher;
+import org.junit.jupiter.api.function.Executable;
+
+/**
+ * Helper assertion methods for verifying exceptions.
+ */
+public class ExceptionAssertions {
```

Comment:
Ooh, when I saw it I thought it was a cool Junit method I could use too. But nice idea. Makes the tests more readable.